roles/aws/admin-route53/README.md

```diff
@@ -0,0 +1,68 @@
+Role Name
+=========
+
+Manage Route 53 Zones
+
+Requirements
+------------
+
+The following packages are required:
+
+  - python >= 2.6
+  - boto
+
+The following ENV variables associated with your AWS account must exist to use the module:
+
+  - AWS_ACCESS_KEY_ID
+  - AWS_SECRET_ACCESS_KEY
+
+As an alternative (not recommended for being sensible data) the following ansible variables can be defined as part of the inventory:
+  
+  - aws_access_key
+  - aws_secret_key
+
+Role Variables
+--------------
+
+Two differnt lists can be defined as variables, with another nested list included on each:
+
+  - zone_add --> This list with the nested elements will be used to asure the zone and entries exists on Route 53 DNS service
+  - zone_rm --> This list with the nested elements will be used to asure the zone or the entries do not exist on Route 53 DNS service
+  - zone --> The Domain Name for the Hosted Zone
+  - entries --> This is the list of records to be added/deleted from the parent zone
+  - type --> The type of DNS record
+  - hostname --> Name of the record set
+  - ip --> The value for the record set
```

Comment:
Please make sure to name these as they are named by the module - i.e.: `ip` -> `value`, `hostname` -> `record`. 

We ran into an issue with the `dns` role for the same reason and have an open issue to "fix" it:
https://github.com/redhat-cop/infra-ansible/issues/26

Reply:
done